ssms/config/_modelconfig/base.py

```diff
@@ -60,6 +60,27 @@
             "donset",
         ],
     },
+    "conflict_stimflex_drift": {
+        "fun": df.conflict_stimflex_drift,
+        "params": ["v_t", "v_d", "tcoh", "dcoh", "tonset", "donset"],
```

Comment:
let's just use consistent formatting here with one parameter per line?
if ruff autocorrects that then nvm.